Spectrogram.sc

```diff
@@ -216,6 +216,7 @@ Spectrogram {
 
 	stop { this.stopruntask }
 
+	isRunning { ^if(running == true) { true } { false } }
```

Comment:
This line could just be `^running`.

I meant to leave this as a comment but accidentally pressed review, and don't seem to be able to reverse rhis... I don't know if that is okay or if there is some special privilege or authorisation I should get before doing this? Very unfamiliar with the etiquette around pull requests :)

Reply:
I corrected it according to your suggestion! Thanks! My coding level is very basic, so I did not think this simple solution! This was an enlightening moment for me! Very appreciated!